tests/questionTypes.js

```diff
@@ -161,6 +161,11 @@ define([
                 type: 'AndroidIntent',
                 path: 'question22/question23/',
                 nodeId: 'question7'
+            }, {
+                //clickBeforeAdd: "question32",
```

Comment:
Can you expand on this comment?

Reply:
why did we move the `MicroImage` block up?

Reply:
Ah, that must have slipped through when I was working on this test. Will remove this comment.

> why did we move the MicroImage block up?

This was purely to help make it more readable by putting it in the same position on the test as it is in the `TEST_XML` file.